### DISCUSSION TITLE
Celery Chain not working while using older django

### DISCUSSION THREAD

**Discussion by SnoozeFreddo** [celery/celery#7315] [Q&A]
Hi there,

i am using celery-beat 2.2.1 and celery 5.1.2.

i created a chain task with two ways.
1/ by using celery beat

 `"chained": {
            "task": "proj.tasks.get_or_create",
            "schedule": crontab(minute="*/2"),
            "options": {
                "link": signature(
                    "proj.tasks.get_or_create_list",
                    options={
                        "link": signature("proj.tasks.recover")
                    },
                )
            },
        }`
        
and the 2/ by using:
`chain(
                                get_or_create_folder.si(),
                                get_or_create__list.s(),
                                _import.s(csv),  
                            ).apply_async()`
                            
                            
When using djangorestframework 3.12.4 it works as expected and django 3.2.8 but when i use djangorestframework 3.9.4 and django 2.2.24 it only executes the first task successful without errors but doesn't continue the chain.

What am i doing wrong?

[a comment by an automated account is not shown]

**Comment by thedrow** [comment]
I don't think we have enough information to resolve your problem as of now.
